AzWebAppUP/runtime_detection.md

```diff
@@ -29,6 +29,23 @@ Node detection looks for the existence of the `package.json` or `sever.js` or `i
 
 ASP .NET and .NET Core detection is done by looking for the existence of `*.csproj` in the root folder of the project.
 
+### .NET 5
+
+Detection logic for .NET 5 apps will look into your `*.csproj` file in the current folder.
+
+Detection will then check value set for  `TargetFramework` element looking for `net5.0` string.
```

Comment:
FYI @panchagnula and @calvinsID 

Keep me honest here

Reply:
Yup looks good